lib/src/folding/parsers/fallback.dart

```diff
@@ -99,30 +113,94 @@ class FallbackFoldableBlockParser extends TextFoldableBlockParser {
                 }
                 break;
               }
+              if (c.contains(char)) {
+                _isPossibleCommentSequence = true;
+              }
+            }
+
+            if (isMultilineCommentingEnabled) {
+              for (final c in multilineCommentSequences!) {
+                if (tail.endsWith(c.item1)) {
+                  if (!serviceCommentLines.contains(lineIndex)) {
+                    if (!foundImportTerminator) {
+                      startBlock(lineIndex, FoldableBlockType.multilineComment);
+                      _shouldEndMultilineComment = true;
+                    } else {
+                      // shouldn't start multiline comment block
+
+                      // class MyClass { /*
+                      // */
+                      // }
```

Comment:
Check if this can be simplified if we make the highlight parser return union for 
```
{
} /*
*/
```